circuits/src/zk_gadgets/wallet_operations.rs

```diff
@@ -2,6 +2,7 @@
 //! circuit
 
 use circuit_types::{
+    balance::BalanceVar,
```

Comment:
there's a bunch of this, I'm going to clean them up later, my rust-analyzer won't show them because of other errors in the crate. If you're good to review around clippy lints for the moment that'd be appreciated.